dts/arm/st/l5/stm32l5.dtsi

```diff
@@ -337,6 +337,15 @@
 			status = "disabled";
 		};
 
+		sdmmc1: sdmmc@420c8000 {
+			compatible = "st,stm32-sdmmc";
+			reg = <0x420c8000 0x400>;
+			clocks = <&rcc STM32_CLOCK_BUS_AHB2 0x00400000>;
+			interrupts = <78 0>;
+			status = "disabled";
+			label = "SDMMC_1";
```

Comment:
Labels should now be removed.
cf https://github.com/zephyrproject-rtos/zephyr/pull/48358